tests/components/input_datetime/test_reproduce_state.py

```diff
@@ -36,7 +36,11 @@
 
     # Test invalid state is handled
     await hass.helpers.state.async_reproduce_state(
-        [State("input_datetime.entity_datetime", "not_supported")], blocking=True
+        [
```

Comment:
I'm not familiar with `reproduce_state`, but given that you're adding an entry to the list, shouldn't you check that this is handled correctly? Something like:

```python
state = hass.states.get("input_datetime.entity_datetime")
assert state is not None
assert state.state == "whatever the correct state should be"
```

Reply:
That is not necessary here since we count the calls. We try to reproduce something invalid here so we expect zero calls: 
```python
    assert "not_supported" in caplog.text
    assert len(datetime_calls) == 0
```

Reply:
Gotcha. I didn't see the `len(datetime_calls)` line, it got cutoff in the "Files changed" view.